src/sections/Home/Partners-home/partners-home-data.js

```diff
@@ -29,12 +29,12 @@ export const partners = [
   },{
     name: "Citrix",
     imageLink: citrix, 
-    imageRoute: "/partners#utaustin",
+    imageRoute: "/partners#Citrix",
     innerDivStyle: "partner__block__inner"
   },{
     name: "Hashicorp partnership with Layer5",
     imageLink: hashicorp, 
-    imageRoute: "/company/news/layer5-and-hashicorp-launch-service-mesh-partnership",
+    imageRoute: "/partners#HashiCorp",
```

Comment:
but it is working fine even after changing, so I thought it may not affect any workflow, and I just changed that to make code even for all, I am not sure if it brings any conflict or not 😅

Reply:
Yes @Abhijay007 could you please revert this change. The `Hashicorp` logo is supposed to redirect to the news article of hashicorp and not to the partners page.

Reply:
Before ,it opened other page https://layer5.io/company/news/layer5-and-hashicorp-launch-service-mesh-partnership ,  which have more information, so think it will be better to redirect to that page

Reply:
ok